process/start_project.md

```diff
@@ -0,0 +1,107 @@
+
+### Leverage the collective experience and expertise of the Linux Foundation.
+
+Academy Software Foundation staff collectively have decades of experience driving open-source projects and ecosystems in various vertical and horizontal industries, bringing the best practices to your project.
+
+- Press relations and analyst relations teams to increase awareness of and excitement about your project, and receive regular reports on press mentions and comparisons to similar projects.
+- Academy Software Foundation can leverage the expertise of its parent organization, The Linux Foundation, which hosts other successful projects, including the Linux kernel, Let’s Encrypt, and Node.js. 
+- You will have access to full-time Academy Software Foundation staff eager to assist your project in myriad ways and help make it successful.
+
+## Preparing to bring your project here
+
+The “LF Way” is that there is no one way. The LF assumes that each open source project is different in its own way, which leads to building a unique governance and solution to sustain their communities.
+
+### Requirements to host a project here
+
+The project must:
+
+- Use an approved [OSI open-source license](https://opensource.org/licenses).
+- Be supported by a [TAC member](https://www.aswf.io/about/technical-advisory-council/).
+- Allow neutral ownership of project assets such as a trademark, domain, or GitHub account (the community can define and manage rules).
+- Technical "do-ocracy" and separation of business governance from technical governance (we're flexible on the model) are clearly documented (which we can help you establish).
+- Allow anyone to participate in the technical community, whether or not a financial member or supporter of the project.
+
+Often projects are new open-source projects, but that isn’t a requirement - existing projects are welcome!
+
+### Project Preparation Checklist
+
+- Prepare draft mission and scope statement. 
+- Identify internal teams within the mission and scope. 
+- Identify developers internally 
+- Analyze code for identification of licenses, license quality, and project dependencies 
+- Assess the potential universe of project participants. Identify ecosystem constituents and concerns of each. 
+- Identify any trademarks leveraged by the project (these will be owned by the LF neutrally) 
+- Outline infrastructure, marketing any other requirements needed for the project. 
+- Complete the [project proposal](https://tac.aswf.io/process/proposal_template) and submit it to the TAC for consideration
+
+The Academy Software Foundation staff is happy to engage and advise during this process - let us know how we can help!
+
+### What happens next?
+
+Generally, the TAC will schedule project proposals at the following [TAC Meeting](/meetings). Guidelines for the presentation to the TAC are detailed in the [project lifecycle](/process/lifecycle#tac-presentation).
```

Comment:
Is this new text? It doesn't seem to have been moved from anywhere else, just wanted to know how closely it needs to be reviewed if it is new (or presented to TAC?).

Reply:
Yes - and yes I would love TAC feedback on this.